website/blog/authors.yml

```diff
@@ -0,0 +1,17 @@
+KenC:
+  name: Ken Chen
+  title: Principal Software Engineer
```

Comment:
Ken is manager ...

Reply:
Since the title will be visible to the public, my thinking was that it's important to show the technical role a team member plays than the managerial role. What do you think?

Reply:
I just found other microsoft dev websites simply display the actual title, https://devblogs.microsoft.com/dotnet/author/kenchen/

Reply:
Got it. Definitely need to follow the same practice across Azure. It's updated already.